contrib/kube-prometheus/manifests/grafana-dashboardDefinitions.yaml

```diff
@@ -1703,7 +1703,7 @@ data:
                         "steppedLine": false,
                         "targets": [
                             {
-                                "expr": "1 - sum(max by (device, node) (node_filesystem_free{fstype=\u007e\"ext[24]\"})) / sum(max by (device, node) (node_filesystem_size{fstype=\u007e\"ext[24]\"}))",
+                                "expr": "1 - sum(max by (device, node) (node_filesystem_avail{fstype=\u007e\"ext[24]\"})) / sum(max by (device, node) (node_filesystem_size{fstype=\u007e\"ext[24]\"}))",
```

Comment:
not happy with this, I think we should probably check in the lock file for the generated manifests, let's move forward with this for this PR though and then make sure to add the lock file in a consecutive PR